README.md

```diff
@@ -31,22 +31,31 @@ Module and runtime structures:
 
 ![Dependency Graph](docs/images/dependency-graph.png)
 
-Modules:
+#### Modules:
 
-- command > all command side code
-- events > event definitions
-- query >
-  - query processor to update the database projection
-  - query controllers and query features
+- auth: Module to handle authentication.
+- command:
+    - Command controllers and features.
+    - Command, and handlers which generate events.
+- core: Event definitions, and supporting classes.
+- event-store: Interface to Eventstore.
+- health: Functionality to determine the application health.
+- query:
+  - Query processor to update the MongoDB projection.
+  - Query controllers and features.
+- user: Module to administrate the application users.
 
-The runtime is defined by four Docker containers / `Dockerfile`s. In case of scaling up or out, the `query processor` should be put in its own container (probably stays as a single container).
+The runtime is defined by multiple Docker containers / `Dockerfile`s. In case of scaling up or out, the `query processor` should be put in its own container (probably stays as a single container).
 
 ### Usage
 
 #### Standalone
 
 Eventstore:
-* Windows: EventStore.ClusterNode.exe --db ./db --log ./logs
+* Should be running. For instructions, view the [Eventstore Documentation](https://developers.eventstore.com/).
+
+MongoDB:
+* Should be running. For instructions, view the [MongoDB Documentation](https://docs.mongodb.com/manual/installation/).
 
 Backend App:
 * npm install
```

Comment:
`npm ci` is beter, toch? Dan wordt er gebruik gemaakt van de bestaande `package-lock.json` ipv dat er een nieuwe wordt gegenereerd ...